azure-pipelines.yml

```diff
@@ -682,7 +682,7 @@ jobs:
 
 - job: TestYumPackage
   displayName: Test Yum Package
-
+  timeoutInMinutes: 120
```

Comment:
Why submit this change?

Reply:
Yea, I merged with master instead of dev. Coming back to the PR today, and working on the fix.

Reply:
The merge issues should be resolved, and the PR should now only contain the desired changes :)